dirty_cat/super_vectorizer.py

```diff
@@ -95,12 +95,26 @@ class SuperVectorizer(ColumnTransformer):
     ----------
 
     cardinality_threshold : int, default=40
-        Two lists of features will be created depending on this value: strictly
-        under this value, the low cardinality categorical values, and above or
-        equal, the high cardinality categorical values.
-        Different transformers will be applied to these two groups,
-        defined by the parameters `low_card_cat_transformer` and
+        Cardinality threshold for categorical (string) features.
+        Two lists of features will be created depending on this value: between
+        3 (included) and this value (excluded), the low cardinality categorical
+        features, and above or equal, the high cardinality categorical features.
+        Different transformers will be applied to these two groups, defined by
+        the parameters `low_card_cat_transformer` and
         `high_card_cat_transformer` respectively.
+        The features with a cardinality of <= 2 will be binary encoded
+        (see `binary_cat_transformer`).
+
+    binary_cat_transformer : typing.Optional[typing.Union[sklearn.base.TransformerMixin, typing.Literal["drop", "remainder", "passthrough"]]], default=None  # noqa
```

Comment:
Why do we need a special transformer for binary categories? Can we not simply add the option 'drop="if_binary"' to the transformer used in low_card_cat_transformer. It makes the interface and the code much simpler.